src/solvers/prop/prop.cpp

```diff
@@ -38,8 +38,15 @@ bvt propt::new_variables(std::size_t width)
 
 propt::resultt propt::prop_solve()
 {
+  static const bvt empty_assumptions;
   ++number_of_solver_calls;
-  return do_prop_solve();
+  return do_prop_solve(empty_assumptions);
```

Comment:
How about just using `{}` as an argument here instead of the `static` local?

Reply:
The idea is that the static prevents the creation of a new object, and hence, would be a tiny bit more efficient. But that may well not be measurable.